lib/matplotlib/animation.py

```diff
@@ -416,22 +415,10 @@ def bin_path(cls):
     @classmethod
     def isAvailable(cls):
         '''
-        Check to see if a MovieWriter subclass is actually available by
-        running the commandline tool.
+        Check whether the command-line tool for a MovieWriter subclass actually
+        exists in the PATH.
         '''
-        bin_path = cls.bin_path()
```

Comment:
This is here to let sub-classes apply some extra filtering, please revert this change.